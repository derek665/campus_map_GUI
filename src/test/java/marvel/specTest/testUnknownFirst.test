# test correct output when the start node is unknown

LoadGraph testFirst soccer.tsv

FindPath testFirst Chelsea Liverpool