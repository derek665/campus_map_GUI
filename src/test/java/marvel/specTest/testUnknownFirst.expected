# test correct output when the start node is unknown

loaded graph testFirst

unknown character Chelsea